### ISSUE TITLE
Suggest to change "camelCase" column name  to "camel_case"

### ISSUE THREAD

**Issue by ihipop** [yii2mod/yii2-settings#11]
https://github.com/yii2mod/yii2-settings/blob/0c9160253dc1149b43b9d8d3723d462bf691081d/migrations/m150227_114524_init.php#L28

1. In Yii2's `TimestampBehavior` ,Default `$createdAtAttribute` is `camel_case` style

1. `camel_case` style column name is most common  database practise supported by `PDO`

1. when writing a `SQL` by hand,`camel_case` style can keep the consistency of SQL,  to avoid a mixed use of `camelCase` and `camelcase`

1. `TableName` in  `CamelCase` is unacceptable on  `consistency` when deploying among different databases and platforms supported by `PDO`

# This is just a Suggestion

**Comment by ihorchepurnyi** [comment]
Hi, thanks for issue. We have plans for change columns/table-names in all our repos. In the next few days we'll start.

**Comment by ihipop** [comment]
@igor-chepurnoi if there was any `Break Compatibility` change,Suggest to start a new major version
eg. 2.x to 3.x , I use "^2.2" in composer.json  https://getcomposer.org/doc/articles/versions.md#caret

**Comment by ihorchepurnyi** [comment]
ok, thanks for help

**Comment by ihorchepurnyi** [comment]
Resolved by #12

**Comment by ihorchepurnyi** [comment]
Please execute new migration by the following command:
```sh
$ php yii migrate --migrationPath=@vendor/yii2mod/yii2-settings/migrations
```
